scripts/wasmedge-init.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# Init the WasmEdge environment  (with wasi-nn plugin and tf-lite backend)
+
+set -ex
+
+source "$(dirname -- "$0")/env.sh"
+
+export WASMEDGE_VERSION=0.12.0-alpha.2
```

Comment:
Please update to `0.12.1`.

Reply:
The version ```0.12.1``` cannot run the tests, because it will double-free the ```NNGraph``` and cause segment fault. So I use the build from the master branch source code method now.